fiware-region-sanity-tests/resources/scripts/jenkins.sh

```diff
@@ -303,6 +309,18 @@ setup)
 	make
 	make install
 
+    # Update Restart job
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-0-RestartTestServers/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-0-RestartTestServers.xml"
+
+    # Update SetUp job
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-0-SetUp/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-0-SetUp.xml"
+
+	# Update Flow job
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-1-Flow/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-1-Flow.xml"
+
+	# Update Exec job
+	curl -X POST http://$JENKINS_USER:$JENKINS_PASSWORD@127.0.0.1:8080/job/FIHealth-SanityCheck-2-Exec-Region/config.xml --data-binary "@resources/jenkins/FIHealth-SanityCheck-2-Exec-Region.xml"
+
```

Comment:
I think we shouldn't put fixed URLs here, and use `$JENKINS_URL` instead. In fact, this is the enviroment variable we could check at the beginning of the script execution (and not `$JOB_URL`, which is unnecessary).